SIL.Windows.Forms/ImageGallery/ThumbnailViewer.cs

```diff
@@ -41,13 +41,13 @@ public IThumbnailViewer CreateViewer()
 			if (UseWebViewer)
 			{
 				var path = Path.Combine(Path.GetDirectoryName(new Uri(Assembly.GetExecutingAssembly().CodeBase).LocalPath),
-					"PalasoUIWindowsForms.GeckoBrowserAdapter.dll");
+					"SIL.Windows.Forms.GeckoBrowserAdapter.dll");
 				if (File.Exists(path))
 				{
 					var assembly = Assembly.LoadFile(path);
 					if (assembly != null)
 					{
-						var browser = assembly.GetType("Palaso.UI.WindowsForms.ImageGallery.WebThumbnailViewer");
+						var browser = assembly.GetType("SIL.Windows.Forms.GeckoBrowserAdapter.WebThumbnailViewer");
```

Comment:
I dunno. I assumed Damien and company just put WebThumbnailViewer over there for their own reasons. (WebThumbnailViewer IS a GeckoFx control, as far as I understand things). I didn't move it, I just figured out where it had gotten to. :)